labs/SettingsControls/src/SettingsCard/SettingsCard.xaml

```diff
@@ -168,27 +132,115 @@
                                 <RowDefinition Height="Auto" />
                             </Grid.RowDefinitions>
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="Auto" MinWidth="{ThemeResource SettingsCardLeftIndention}" />
                                 <ColumnDefinition Width="*" />
                                 <ColumnDefinition Width="Auto" />
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <ContentPresenter
```

Comment:
I think your local XAML Styler settings got flipped, there shouldn't be as many changes here? Try running the script on the commandline as the CI is failing?